client/src/components/LoginSession/tests/LoginSession-test.js

```diff
@@ -40,22 +42,52 @@ describe('LoginSession', () => {
             LastAccessed: '2021-03-11 03:47:22',
             LogOutEndpoint: 'admin/loginsession/remove',
         };
+        wrapper = shallow(<Component {...props} />);
+        button = wrapper.find('.login-session__logout');
+    });
+    it('should display details', () => {
+        const html = wrapper.html();
+        expect(html.indexOf('Chrome on Mac OS X 10.15.7')).not.toBe(false);
+        expect(html.indexOf('127.0.0.1')).not.toBe(false);
+        expect(html.indexOf('Signed in 01/20/2021 12:33 AM')).not.toBe(false); // TODO: is this date supposed to have US formatting?
+        expect(html.indexOf('Last active 03/11/2021 3:47 AM')).not.toBe(false); // TODO: is this date supposed to have US formatting?
```

Comment:
I've raised a follow up issue to look into this, so I think we can leave the TODO's in on this PR

https://github.com/silverstripe/silverstripe-session-manager/issues/41